tests/framework/common.go

```diff
@@ -1546,3 +1549,19 @@ func (td *OsmTestData) GrabLogs() error {
 
 	return nil
 }
+
+// AddOpenShiftSCC adds the specified SecurityContextConstraint to the given service account
+func (td *OsmTestData) AddOpenShiftSCC(scc, serviceAccount, namespace string) error {
+	if !td.DeployOnOpenShift {
+		return errors.Errorf("Tests are not configured for OpenShift. Try again with -deployOnOpenShift=true")
+	}
+	//oc adm policy add-scc-to-user privileged -z bookbuyer -n "$BOOKBUYER_NAMESPACE"
+	args := []string{"adm", "policy", "add-scc-to-user", scc, "-z", serviceAccount, "-n", namespace}
```

Comment:
In a subsequent change, let's use their Go SDK instead so we don't introduce a dependency on a `oc cli` here